### DISCUSSION TITLE
Attivate le discussioni a livello di organizzazione

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/comunita#4] [Annunci]
Da [pochi giorni](https://github.blog/changelog/2022-04-12-organization-discussions/) github offre questa opportunità. Noi avevamo già attivato la cosa su un repo, ma adesso visibile a livello di organizzazione.

![image](https://user-images.githubusercontent.com/30607/163672186-c91ba4b9-4083-4808-b260-6b771a586d85.png)

**Comment by dennisangemi** [comment]
Grazie @aborruso! Secondo te le sfrutteremo? Per cosa potremmo utilizzarle?

**Reply by aborruso** [reply]
Caro @dennisangemi dipende da noi. Per il momento non lo sa quasi nessuno e non c'è un flusso immaginato per farne uso.

È uno strumento che a me piace molto.

**Comment by cirospat** [comment]
la funzionalità è comoda,
secondo me serve un input per cominciare ad usarla

Buona Pasqua




Ciro Spataro

***@***.***
<https://www.linkedin.com/in/cirospataro/>
<https://www.twitter.com/cirospat>  ***@***.***>
<https://cirospat.readthedocs.io/>





Il giorno sab 16 apr 2022 alle ore 22:36 Andrea Borruso <
***@***.***> ha scritto:
